ONSdigital/ras-party
# Temporarily suspend delete cronjob and bump requests from 2.30.0 to 2.31.0

# What and why?

- temporarily suspend the deletion cronjobs during deployment
- bump requests from 2.30.0 to 2.31.0

# How to test

- deploy to an env and ensure the `party-scheduler-delete-respondents` cronjob is suspended when deployed (using `/deploy` in this PR may not deploy the PR's tgz of this updated Helm chart change, one alternative is to manually deploy it using Helm from a local branch)
- run the acceptance tests to ensure the bump requests to 2.31.0 is ok
